packages/phone-sensors/README.md

```diff
@@ -17,15 +17,28 @@ ns plugin add @awarns/phone-sensors
 
 ## Usage
 
-After installing this plugin, you'll have access to two task groups to start and stop the data collection process.
+After installing this plugin, you'll have access to two task groups with two tasks each to start and stop the data collection process.
+The main difference between both tasks groups is the implementation of the underlying service that being used for the data collection.
```

Comment:
I think something is missing in this sentence. "that ***is*** being used?"

Reply:
Yep, that was missing